Functions/xASL_quant_Basil.m

```diff
@@ -59,21 +59,18 @@
     % FIXME would be good to have a brain mask at this point -> PM: if this would be a brainmask as well, we can skip creating a dummy input image here
     PWI(isnan(PWI)) = 0;
     
-    if size(PWI,4) == 1 
+    if (x.modules.asl.bTimeEncoded == 0) && (x.modules.asl.bMultiPLD == 0)
```

Comment:
Probably a stupid question, but we are sure that those sub-fields are always defined, right?

Reply:
No - a very important question! Yes, they are always defined at the start of ASL_module, so that we don't have to check all the time if the field exist. These fields are not defined in dataPar, but rather derived at the start of the module from the data. Good that you check these things...

Reply:
Currently, we have multiPLD and TimeEncoded as two different things. I'm fine to redefine this and change this as TimeEncoded sub-case of multiPLD. But lets discuss this - note that these are internal parameters.

The second point. Makes sense to always save as PWI4D, but I didn't write this if-statement. Shall we completely remove it then?

Reply:
We discussed this multiple times, and I have always emphasized that bMultiPLD should be true when bTimeEncoded is true. So TimeEncoded is a special case of multiPLD, this is to make life simpler. Throughout the pipeline, both multiPLD and TimeEncoded do exactly the same, except for a few locations only — e.g., when you decode Hadamard volumes into multiPLD volumes.

The change is probably small: we should:
1) verify at the start of the ASL module that this is the case
2) any time that you request bMultiPLD OR bTimeEncoded, you remove bTimeEncoded

Yes, I think I wrote this if-statement because it didn't work with PWI4D or something similar. If this is easy changed it would be good, otherwise we can do this later.

Most important probably is to do the comparison with Martin Craig.

Reply:
Fixed bTimeEncoded being always bMultiPLD a10bfd4